index.html

```diff
@@ -4,9 +4,27 @@
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+  <body>
+    <header class="page-header">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item">
+         <a class="navigation-link" href=about.html">Каталог </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Каталог </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="faq.html">Каталог </a>
+          <main class="main-index">
+            <h1  class="visually-hidden"> Проект «Седона»  </h1>
```

Comment:
Лишние пробелы в тексте нужно убрать:
```suggestion
            <h1  class="visually-hidden">Проект «Седона»</h1>
```